qcut/packages/license-server/.env.example

```
@@ -21,5 +22,15 @@ STRIPE_TOPUP_MEGA_PRICE_ID=price_xxx
 # Mock mode: set to "true" to bypass Stripe/Supabase (local dev)
 MOCK_MODE=false
 
+# Payment runtime controls
+PAYMENTS_WEB_BASE_URL=https://quriosity.com.au
+PAYMENTS_CHECKOUT_ENABLED=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PAYMENTS_CHECKOUT_ENABLED key should go before the PAYMENTS_WEB_BASE_URL key

(UnorderedKey)